python_modules/dagster/setup.py

```diff
@@ -138,7 +138,7 @@ def get_version() -> str:
             "morefs[asynclocal]",
             "fsspec<2024.5.0",  # morefs incompatibly
             "rapidfuzz",
-            "requests<2.32.0",  # 2.32.0 breaks our docker tests https://buildkite.com/dagster/dagster-dagster/builds/83562
+            "requests!=2.32.0",  # https://github.com/dagster-io/dagster/pull/21977
```

Comment:
do we need it here at all with the pin in place in `dagster-docker`?

Reply:
`[tests]` installs `docker` directly